README.md

```diff
@@ -69,9 +69,9 @@ At most, 5 kube-state-metrics and 5 [kubernetes releases](https://github.com/kub
 | kube-state-metrics | **Kubernetes 1.19** | **Kubernetes 1.20** | **Kubernetes 1.21** | **Kubernetes 1.22** | **Kubernetes 1.23** |
 |--------------------|:-------------------:|:-------------------:|:-------------------:|:-------------------:|:-------------------:|
 | **v1.9.8**         |          -          |          -          |          -          |          -          |          -          |
```

Comment:
We probably should remove 1.9.x at this point as well?

Reply:
I would personally be in favor of removing v1.9.8 instead of v2.1.1. None of the Kubernetes versions that are currently supported by kms are supported by v1.9.8, so I don't think it is worth continuing supporting it.

Reply:
Makes sense, removed 1.9.8